docker/requirements.txt

```diff
@@ -0,0 +1,4 @@
+six>=1.16.0  # required by ma5
+scipy>=1.7.0  # optional for reinterpretation
+pyhf>=0.6.3  # optional for reinterpretation
+matplotlib>=3.5.0  # optional for histogramming
```

Comment:
Including all of these as they are relatively lightweight dependencies.

Reply:
this is perfect thanks I can add more as needed! also do you know if the latex and pdflatex compilers should be downloaded separately? I guess if you managed to execute the example they are already in the machine by default?